piyushgarg-dev/piyushgargdev-nextjs
# fix: [Fix #59] Implement mobile navbar close menu

## What does this PR do?

This PR implements the mobile nav menu which was earlier notshowing.

Fixes #59

BEFORE=>
![image](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/110960043/88690b11-9be4-459f-ac1f-b4158e4bc62a)

AFTER=>
![image](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/110960043/2208ac2e-285b-424b-9537-5036692665fd)


## How should this be tested?

1. Open the website locally http://localhost:3000/
2. Go To inspect element and decrease the screen size when hamburger menu is visible.
3. click on the hamburger menu.
4. You will see close menu on the top right of the screen


Thank You
